extensions/panache/mongodb-rest-data-panache/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,15 @@
+---
+name: "REST resources for MongoDB with Panache"
+metadata:
+  keywords:
+  - "mongodb-panache"
+  - "panache"
+  - "mongodb"
+  - "rest"
+  - "jaxrs"
+  - "resteasy"
+  guide: "https://quarkus.io/guides/rest-data-panache"
```

Comment:
Will there be a single doc for both Hibernate and Mongo REST Data ?

Reply:
Initially the idea was to have a single guide because the API is basically the same. I haven't updated the guide yet, I plan to do it separately once we merge the extension.